WordPress/UITests/Tests/DashboardTests.swift

```diff
@@ -39,4 +39,15 @@ class DashboardTests: XCTestCase {
             .tapPagesCardHeader()
             .verifyPagesScreenLoaded()
     }
+
+    func testActivityLogCardHeaderNavigation() throws {
+        try MySiteScreen()
+            .scrollToActivityLogCard()
+            .verifyActivityLogCard()
+            .verifyActivityLogCard(hasActivityPartial: "Enabled Jetpack Social for")
+            .verifyActivityLogCard(hasActivityPartial: "The Jetpack connection is")
+            .verifyActivityLogCard(hasActivityPartial: "This site is connected to J")
```

Comment:
while what's displayed on the simulator might be partial (depending on which simulator is used), the string is displayed in full in the element tree:
<img width="847" alt="image" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/17252150/c7f1c9bd-e68e-4f43-852b-517bcb1faa98">

what do you think about using the full text instead, especially for the last one seems a bit to odd to end with `to J` 

I tried to find where we could add the `accessibilityIdentifier` (which is ideal, so we don't have to care what the text is) but couldn't figure it out...